src/LiveSplit.Core/Server/CommandServer.cs

```diff
@@ -291,6 +294,94 @@ private void ProcessMessage(string message, IConnection clientConnection)
+                    {
+                        region = State.Run.Metadata.RegionName;
+                    }
+                }
+                else if (State.Run.Metadata.Region != null && !string.IsNullOrEmpty(State.Run.Metadata.Region.Abbreviation) && State.Run.Metadata.Game != null && State.Run.Metadata.Game.Regions.Count > 1)
+                {
+                    region =  State.Run.Metadata.Region.Abbreviation;
+                }
+                //Platform
+                string platform = null;
+                bool doSimplePlatform = !State.Run.Metadata.GameAvailable;
+                if (!string.IsNullOrEmpty(State.Run.Metadata.PlatformName) && (doSimplePlatform || (State.Run.Metadata.Game != null && State.Run.Metadata.Game.Platforms.Count > 1)))
+                {
+                    if (State.Run.Metadata.UsesEmulator)
+                    {
+                        platform = $"{State.Run.Metadata.PlatformName} Emulator";
+                    }
+                    else
+                    {
+                        platform = State.Run.Metadata.PlatformName;
+                    }
+                }
+                else if (State.Run.Metadata.UsesEmulator)
+                {
+                    platform = "Emulator";
+                }
+                //Variables
+                Dictionary<string, string> varDict = new Dictionary<string, string>();
+                IEnumerable<string> variables = State.Run.Metadata.VariableValueNames.Keys;
+                if (State.Run.Metadata.GameAvailable && State.Run.Metadata.Game != null)
+                {
+                    string categoryId = null;
+                    if (State.Run.Metadata.CategoryAvailable && State.Run.Metadata.Category != null)
+                    {
+                        categoryId = State.Run.Metadata.Category.ID;
+                    }
+                    
+                    variables = State.Run.Metadata.Game.FullGameVariables.Where(x => x.CategoryID == null || x.CategoryID == categoryId).Select(x => x.Name);
+                }
+
+                foreach (string variable in variables)
+                {
+                    if (State.Run.Metadata.VariableValueNames.ContainsKey(variable))
+                    {
+                        string name = variable.TrimEnd('?');
+                        string variableValue = State.Run.Metadata.VariableValueNames[variable];
+                        string valueLower = variableValue.ToLowerInvariant();
+                        if (valueLower == "yes")
+                        {
+                            varDict.Add(variable, name);
+                        }
+                        else if (valueLower == "no")
+                        {
+                            varDict.Add(variable, $"No {name}");
+                        }
+                        else
+                        {
+                            varDict.Add(variable, variableValue);
+                        }
+                    }
```

Comment:
```suggestion
                    if (State.Run.Metadata.VariableValueNames.TryGetValue(variable, out string value))
                    {
                        varDict.Add(variable, value);
                    }
```

Is the special handling for `Yes` and `No` really necessary?

Reply:
Without them, the values for the variable would be displayed as yes or no just like the title component. I don't think there's a way around this scenario all together, but at least the key will have the variable name for the command so I'll do away with it.